[Torch uplift] RecursionError in MiniCPM loader monkey patch (`nn.Module.__getattr__`)

Multiple models are failing with `RecursionError` in MiniCPM loader monkey patch with PyTorch 2.11.0 uplift.
[CI link] (https://github.com/tenstorrent/tt-xla/actions/runs/26130680305)

Few of the failing tests
- tests/runner/test_models.py::test_all_models_torch[phi3/phi_3_5/pytorch-Mini_Instruct-single_device-inference]
- tests/runner/test_models.py::test_all_models_torch[qwen_3/causal_lm/pytorch-4B-single_device-inference]
- tests/runner/test_models.py::test_all_models_torch[phi3/causal_lm/pytorch-Mini_128K_Instruct-single_device-inference]
- tests/runner/test_models.py::test_all_models_torch[yolov7/pytorch-Default-single_device-inference]

## Observed Failure
```text
E   torch._dynamo.exc.InternalTorchDynamoError: RecursionError: maximum recursion depth exceeded
E   
E   from user code:
E      File "/__w/tt-xla/tt-xla/venv/lib/python3.12/site-packages/transformers/modeling_rope_utils.py", line 127, in torch_dynamo_resume_in_wrapper_at_126
E       return rope_forward(self, x, position_ids, **kwargs)
E     File "/__w/tt-xla/tt-xla/venv/lib/python3.12/site-packages/transformers/models/phi3/modeling_phi3.py", line 121, in forward
E       inv_freq_expanded = self.inv_freq[None, :, None].float().expand(position_ids.shape[0], -1, 1).to(x.device)
E     File "/__w/tt-xla/tt-xla/third_party/tt_forge_models/minicpm_o_2_6/pytorch/loader.py", line 48, in patched_getattr
E       return original_getattr(self, name)
E     File "/__w/tt-xla/tt-xla/third_party/tt_forge_models/minicpm_o_2_6/pytorch/loader.py", line 48, in patched_getattr
E       return original_getattr(self, name)
E     File "/__w/tt-xla/tt-xla/third_party/tt_forge_models/minicpm_o_2_6/pytorch/loader.py", line 48, in patched_getattr
E       return original_getattr(self, name)
E     [Previous line repeated 260 more times]
```

## Root Cause
`loader.py` globally patches `nn.Module.__getattr__` and then delegates to a saved `orig` function. This is fragile because:

1. It mutates a global method used by all Torch modules.
2. If `orig` is captured after any prior patch (or if patch ordering changes across imports), delegation can point to a patched function and recurse.
3. The patch is not limited to MiniCPM/Resampler instances.

--- a/minicpm_o_2_6/pytorch/loader.py
+++ b/minicpm_o_2_6/pytorch/loader.py
@@ -33,23 +33,6 @@ if ALL_PARALLEL_STYLES is None:
 
     mu.ALL_PARALLEL_STYLES = ["rowwise", "colwise", "headwise"]
 
-# Monkey patch Resampler for compatibility - Fixes: Resampler doesn't have _initialize_weights method in torch 2.7.0
-original_getattr = nn.Module.__getattr__
-
-
-def patched_getattr(self, name):
-    if name == "_initialize_weights" and self.__class__.__name__ == "Resampler":
-
-        def _initialize_weights(module_self):
-            if hasattr(module_self, "_init_weights"):
-                module_self._init_weights(module_self)
-
-        return _initialize_weights
-    return original_getattr(self, name)
-
-
-nn.Module.__getattr__ = patched_getattr
-
 
 @dataclass
 class MiniCPMConfig(ModelConfig):

